docs/fragments/README.md

```diff
@@ -0,0 +1,11 @@
+# Fragment Documentations
+
+Syna comes with a set of prebuilt fragments. These fragments are just to make developing a website using Syna easier. It's possible to create new fragments or override the default fragments.
+
+You can add other fragments and override the default fragments using Hugo's [Template lookup order](https://gohugo.io/templates/lookup-order/) strategy. You can create a new file in your website's `/layouts/partials/fragments` and name it after the fragment you want to override or the fragment you want to create and develop the fragment yourself. Although we don't recommend overriding the default fragments since that would cause you to miss the updates.
+
+The following is the documentation of the default fragments. To understand how fragments work please read our guide to [getting started](https://github.com/okkur/syna/tree/master/docs/README.md#fragments).
+
+## Fragments
+
+- [Contact fragment](https://github.com/okkur/syna/tree/master/docs/fragments/contact.md)
```

Comment:
I think the link to the fragment isn't working. it returns 404 page.

Reply:
Yeah the link is pointing to the master branch. I'm not sure how to link to files relatively and make sure Github understands them. I'll change it right away to relative and check it out.